### PR TITLE
Settings 9

### PR DESCRIPTION
**Pull request by markk628**
Pull Request for Issue #9 - Settings

Added Picker objects for unit of measurement and timer

Default ratio storage is saved into UserDefaults

### CODE REVIEW

**Comment by johnmahlon** [comment]
Thanks for the addition! I'll do some code review soon and see about merging it in!

File: Ratios/Ratios/View Models/CalculatorViewModel.swift [hunk 1]
```diff
@@ -13,9 +13,9 @@ typealias UnitOfMeasurement = Double
 class CalculatorViewModel {
     static func calculateUnitOfMeasurementOfWaterTimes(waterRatio: UnitOfMeasurement, coffee: UnitOfMeasurement, unit: Int) -> UnitOfMeasurement {
         switch unit {
-        case 1:
+        case 1: // Tbsp formula
```
Comment (NateFuller) [inline Ratios/Ratios/View Models/CalculatorViewModel.swift:16]: Maybe create a separate issue to refactor `unit` to be an enum type so you don't need to add comments for context?

File: Ratios/Ratios/Views/TimerView.swift [hunk 2]
```diff
@@ -35,7 +35,7 @@ struct TimerView: View {
         }
         
         switch timerType {
-        case 0:
+        case 0: // Count up Timer
```
Comment (NateFuller) [inline Ratios/Ratios/Views/TimerView.swift:38]: Same here. TimerType could be an enum value as well.

**Comment by johnmahlon** [comment]
@markk628 Finally getting around to checking this out. My bad. 

I'm seeing issues with the count down timer. When typing in 60, it will do 1:59 -> 0:58 -> 0:57.. etc. What's the expected way to use the count down timer?

**Pull request closed without merging**